tests/save.js

```diff
@@ -11,14 +11,15 @@ mongoose.plugin(beautifulValidation);
 /**
  * Assert that the given object is a correct validation error.
  *
- * @param {Object} assert Tape assertion object
+ * @param {Object} t Tape assertion object
```

Comment:
Actually, renaming the assertion object is quite a good idea. It makes the tests more terse. Thanks!